src/api/app/controllers/webui/repositories_controller.rb

```diff
@@ -241,12 +242,27 @@ def remove_flag
       @main_object.store
       format.html { redirect_to(action: :index, project: params[:project], package: params[:package]) }
       format.js do
-        switch_to_webui2
         render 'change_flag'
       end
     end
   end
 
+  # POST flag/manipulate/:project(/:package)
+  # TODO when removing bento, remove the extra manipulate from the route, for
+  # now we need to avoid the clash with create_flag
+  def manipulate_flag
```

Comment:
`change_flag`? :thinking:

Reply:
the problem in both cases is: how to diff between :project/:package and manipulate/:project